corehq/apps/linked_domain/updates.py

```diff
@@ -89,7 +88,17 @@ def update_custom_data_models(domain_link, limit_types=None):
 
     for field_type, field_definitions in master_results.items():
         model = CustomDataFieldsDefinition.get_or_create(domain_link.linked_domain, field_type)
-        model.fields = [CustomDataField.wrap(field_def) for field_def in field_definitions]
+        model.sqlfield_set.all().delete()
+        model.sqlfield_set.set([
```

Comment:
This could be DRYer since similar thing is done in one or more places.

Reply:
4f670fada492858f8972d089611bf9d538ff8810 (added to the previous PR, https://github.com/dimagi/commcare-hq/pull/27277)